templates/style/rustdoc-common.scss

```diff
@@ -101,6 +101,10 @@ div.rustdoc {
         }
     }
 
+    .mobile-topbar {
```

Comment:
Since it's the same as the rule following it, maybe merge them?

Reply:
Done!